### ISSUE TITLE
Companion Map

### ISSUE THREAD

**Issue by radius314** [bmlt-enabled/crouton#52]
A nice add-on would be to allow someone (from the admin UI) supply a google javascript api key.  This key would allow for taking the visible results of what's the tabs and display it on a map.  

As you tab through (essentially filtering) the map should update.

Great project for someone who is seasoned but wants to get involved

**Comment by paulnagle** [comment]
I have started something similar to this idea. I needed to write a wordpress plugin to replace some plain javascript code that I had on  https://www.na-ireland.org/na-meetings/meeting-search/ 

So that page is now implemented as a (not published) plugin https://github.com/paulnagle/tabbed_map

It's a WIP, I dont always have a lot of time to spend on it so it'll be a while before its finished. I plan to change from google maps to open street map (so no api key required), also I need to set up a settings page (its all hardcoded for NA Ireland at the moment), and I need to adjust the css for small displays..

When it's done I'd be happy to port the code over to crouton.
